sources/models/src/aws-k8s-1.20/mod.rs

```diff
@@ -0,0 +1 @@
+../aws-k8s-1.19/mod.rs
```

Comment:
Currently we're symlinking the entire aws-k8s-1.19 directory. We can't symlink the entire aws-k8s-1.20 for other variants since aws-k8s-1,20 has it's own overrides (specifically `defaults.d/50-aws-k8s.toml`). If we want to go that route, then we would be symlinking individual files. So for aws-k8s-1.16, aws-k8s-1.17, and aws-k8s-1.18, their `defaults.d/50-aws-k8s.toml` would have to be symlinked to aws-k8s-1.19.

Maybe each variant should just have its own `defaults.d/50-aws-k8s.toml`?

Reply:
Oh, no, I was just thinking at the wrong level, sorry.  No need to duplicate configs on all the old variants.

What I might suggest instead is refactoring 50-aws-k8s.toml so that we have pieces that apply to every variant, and then just some smaller overrides in variants as required.  We could even do a simpler strategy for now, where 1.20 links to the existing 50-aws-k8s.toml, plus has a 51 link to a new lockdown-integrity.toml file.